mathics/doc/common_doc.py

```diff
@@ -679,6 +675,32 @@ def get_section(self, part_slug, chapter_slug, section_slug):
                 return chapter.sections_by_slug.get(section_slug)
         return None
 
+    def get_section_tests(self, section):
```

Comment:
Whereis this called?

Reply:
Ah good. This has been fixed now and you'll see the code deleted from below. 

This was part of the abortive try to get more test results from Mathics3 modules into the LaTeX and Django result output.